src/amplitude-wrapper.js

```diff
@@ -151,14 +153,23 @@ var amplitudeUserAgentEnrichmentPlugin=function(i){"use strict";var e=function()
       };
   };
 
+  var getRegExp = function(expression) {
+    try {
+      return new RegExp(expression);
+    } catch(e) {
+      console.error(`${LOG_PREFIX} - ${e}`);
+      return [];
```

Comment:
Why return an empty array instead of null? You're accumulating a list of regex right? With this you may have [[]] as the final result?

Reply:
Good catch. But return null also doesn't work. Use `reduce` call instead to return the valid not null regular expression.